src/parser/ast/AssignmentExpressionNullishCoalescingNode.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2020-present Samsung Electronics Co., Ltd
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  License along with this library; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301
+ *  USA
+ */
+
+#ifndef AssignmentExpressionNullishCoalescingNode_h
+#define AssignmentExpressionNullishCoalescingNode_h
+
+#include "ExpressionNode.h"
+#include "AssignmentExpressionSimpleNode.h"
+
+namespace Escargot {
+
+class AssignmentExpressionNullishCoalescingNode : public ExpressionNode {
```

Comment:
What is this `AssignmentExpressionNullishCoalescingNode` for?
It seems that nullish coalescing (?? operator) is only applicable for binary expression, not for assignment case. (`??=` operatoration).
Correct me if I'm wrong.